monkey/infection_monkey/exploit/zerologon.py

```diff
@@ -284,14 +289,21 @@ def store_extracted_creds_for_exploitation(self) -> None:
     def send_extracted_creds_as_credential_telemetry(
         self, user: str, lmhash: str, nthash: str
     ) -> None:
-        self.telemetry_messenger.send_telemetry(
-            CredentialsTelem(
-                [
-                    Credentials(Username(user), LMHash(lmhash)),
-                    Credentials(Username(user), NTHash(nthash)),
-                ]
-            )
+        extracted_credentials = [
+            Credentials(Username(user), LMHash(lmhash)),
+            Credentials(Username(user), NTHash(nthash)),
+        ]
+
+        self.telemetry_messenger.send_telemetry(CredentialsTelem(extracted_credentials))
+        self._publish_credentials_stolen_event(extracted_credentials)
+
+    def _publish_credentials_stolen_event(self, extracted_credentials: Sequence[Credentials]):
+        credentials_stolen_event = CredentialsStolenEvent(
+            target=None,
+            tags=(ZEROLOGON_EVENT_TAGS),
```

Comment:
What attack techniques were used to generate this event? It's worth taking a good, long look through the MITRE ATT&CK framework to find them all.

Reply:
We will. For now, let's make sure all relevant tags are included. The report can ignore any it doesn't know how to handle.